runner/src/server/plugins/engine/components/TimeField.ts

```diff
@@ -5,19 +5,30 @@ import { FormComponent } from "./FormComponent";
 import { FormModel } from "../models";
 import { addClassOptionIfNone } from "./helpers";
 import { FormData, FormSubmissionErrors } from "../types";
+import { Schema } from "joi";
 
 export class TimeField extends FormComponent {
   constructor(def: InputFieldsComponentsDef, model: FormModel) {
     super(def, model);
     addClassOptionIfNone(this.options, "govuk-input--width-4");
+    const { options } = def;
+    let componentSchema = helpers.getFormSchemaKeys(this.name, "string", this)[
```

Comment:
both of the getFormSchema and getStateSchema functions were using getFormSchemaKeys, and I think I was worried about moving to a standard string schema in case something broke, so kept using the helper